capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/ui/IntervalToolbarMenuItemIntervalClickListener.kt

```diff
@@ -0,0 +1,24 @@
+package net.gini.android.capture.internal.ui
+
+import android.os.Handler
+import android.os.Looper
+import android.view.MenuItem
+import androidx.appcompat.widget.Toolbar
+
+class IntervalToolbarMenuItemIntervalClickListener(private val itemListener: Toolbar.OnMenuItemClickListener?): Toolbar.OnMenuItemClickListener,
```

Comment:
[Suggestion] Similarly to my other comments we should pass an instance of this to the `NavigationBarTopAdapter.setOnMenuItemClickListener()`.